docker/zap-baseline.py

```diff
@@ -85,21 +88,59 @@ def usage():
     print('    -a                include the alpha passive scan rules as well')
     print('    -d                show debug messages')
     print('    -P                specify listen port')
-    print('    -D                delay in seconds to wait for passive scanning ')
+    print('    -D secs           delay in seconds to wait for passive scanning ')
     print('    -i                default rules not in the config file to INFO')
     print('    -I                do not return failure on warning')
     print('    -j                use the Ajax spider in addition to the traditional one')
     print('    -l level          minimum level to show: PASS, IGNORE, INFO, WARN or FAIL, use with -s to hide example URLs')
     print('    -n context_file   context file which will be loaded prior to spidering the target')
     print('    -p progress_file  progress file which specifies issues that are being addressed')
     print('    -s                short output format - dont show PASSes or example URLs')
-    print('    -T                max time in minutes to wait for ZAP to start and the passive scan to run')
+    print('    -T mins           max time in minutes to wait for ZAP to start and the passive scan to run')
     print('    -U user           username to use for authenticated scans - must be defined in the given context file')
     print('    -z zap_options    ZAP command line options e.g. -z "-config aaa=bbb -config ccc=ddd"')
     print('    --hook            path to python file that define your custom hooks')
+    print('    --auto            use the automation framework if supported for the given parameters (this will become the default soon)')
+    print('    --autooff         do not use the automation framework even if supported for the given parameters')
```

Comment:
I feel that only one of the two options is needed - the automation framework is not used when `--auto` isn't specified, which is the same as `--autooff`, but I may be wrong.

Reply:
This is true at an given time, but the plan is we have a phased approach.
To start with its off, so you will need to use `--auto` to enable. Strictly speaking we wont need `--autooff`.
Then we will switch to being on by default, then you will need `--autooff` to disable it, at which point we can remove `--auto`.
However I added both at the start so that if someone tries it out and finds a problem then they can seet `--autooff` right now and know that their jobs will keep working no matter when we switch over. Then they can retest at their leisure.